uart/lcd_uart/README.adoc

```diff
@@ -1,21 +1,16 @@
+This example code shows how to interface the Raspberry Pi Pico to one of the very common 16x2 LCD character displays. Due to the large number of pins these displays use, they are commonly used with extra drivers or backpacks. In this example, we will use an Adafruit LCD display backpack, which supports communication over USB or TTL. A monochrome display with an RGB backlight is also used, but the backpack is compatible with monochrome backlight displays too. There is <<_attaching_a_16x2_lcd_via_i2c, another example>> that uses I2C to control a 16x2 display.
 
-The backpack processes a set of commands that are documented https://learn.adafruit.com/usb-plus-serial-backpack/command-reference[here] and preceded by the "special" byte 0xFE. The backpack does the ASCII character conversion and even supports custom character creation. In this example, we use the primary UART (uart0) to read characters and print them onto the LCD. We also define a special startup sequence and vary the display's backlight color.
+The backpack processes a set of commands that are documented https://learn.adafruit.com/usb-plus-serial-backpack/command-reference[here] and preceded by the "special" byte 0xFE. The backpack does the ASCII character conversion and even supports custom character creation. In this example, we use the Pico's primary UART (uart0) to read characters and print them onto the LCD. We also define a special startup sequence and vary the display's backlight color.
 
-[NOTE]
-======
-These LCD displays can also be driven directly using GPIO without the use of an adapter board. That is beyond the scope of this example and not very commonly done.
-======
-
-TIP: The Adafruit backpack uses a microcontroller (At90USB162) of its own to interface with commands sent to it and the raw pin outputs, providing a convenient layer of abstraction over the LCD. The backpack breaks out a few pins from this microcontroller and the RX pin too, so you could program it to do more! The software flashed onto the board is also https://learn.adafruit.com/usb-plus-serial-backpack/downloads[available to inspect].
-
-WARNING: While this display will work at 3.3V, it will be quite dim. If powering the Pico via USB (which provides 5V) consider powering the backpack via VBUS (pin 40). However, if you do this, DO NOT connect any other pins on the backpack to the Pico (which is not 5V tolerant) apart from the needed TX (Pico) -> RX (backpack) as the backpack's IO are now at 5V.
+TIP: The Adafruit backpack uses a microcontroller (At90USB162) of its own to interface with commands sent to it and the raw pin outputs, providing a convenient layer of abstraction over the LCD. The backpack breaks out a few GPIO pins from this microcontroller and its TX/RX pins too. The software flashed onto the board is also https://learn.adafruit.com/usb-plus-serial-backpack/downloads[available to inspect].
```

Comment:
I'm still not sure if this is at all relevant to the beginners that this example is aimed at? They probably **don't care** that the backpack uses a microcontroller, and only need to know that you squirt in characters on one side of the board, and get them displayed on the LCD on the other side? :thinking:

Reply:
> They probably don't care that the backpack uses a microcontroller…

Ageed. This just confuses things. It's example code for Pico, not the Adafruit backpack.